src/identity/storage/ExpiringStorageAdapterFactory.ts

```diff
@@ -89,7 +90,10 @@ export class ExpiringStorageAdapter implements Adapter {
 
   public async revokeByGrantId(grantId: string): Promise<void> {
     const grantKey = this.grantKeyFor(grantId);
-    const grants = (await this.storage.get(grantKey) || []) as ResourceIdentifier[];
+    const grants = await this.storage.get(grantKey) as ResourceIdentifier[];
```

Comment:
```suggestion
    const grants = await this.storage.get(grantKey) as (ResourceIdentifier[] | undefined);
```
Is that more correct?

Reply:
True